README.md

```diff
@@ -34,3 +34,14 @@ ARM|Add Default Metric Alerts|Y|N|
 ARM|Add Custom Metric Alerts|Y|N|
 
 
+## Usage Guidelines
+
+### Variable Groups
+
+As maintaining Variable groups in UI is tedious , all AKS pipeline variable groups are managed as code in here.
+
+* To update repository with variable groups if changed through UI , run [Get All Script][variablegroups/scripts/get-all-variable-groups.sh] with a Personal Access Token.
+* Pipeline uses [Update Script][variablegroups/scripts/update-variable-groups.sh] to update all variable groups in [Variable Groups][variablegroups/] while building master branch.
+* To create a new environment, run [Create Script][variablegroups/scripts/add-variable-groups.sh] with access token and environment name to create new variable groups. Fill all fields in the generated files with appropriate values before raising a PR.
```

Comment:
I don't quite understand the point to this, wouldn't it be simpler to just copy the files in source control? then you don't need a PAT locally?

Reply:
I guess one should still want it to run get-all-variable-groups (This should be their personal token though) . But yes, it makes absolutely no sense to use this :) Unlike how i felt last night :)